schemas/client.yml

```diff
@@ -59,34 +59,91 @@
       description: >
         Client domain.
 
+    - name: name
+      level: core
+      type: keyword
+      short: Name of the client system.
+      description: >
+        Name of the client system.
+
+        It can contain what `hostname` returns on Unix systems, the fully
+        qualified domain name, or a name specified by the user. The sender
+        decides which value to use.
+        
+        In the case of fully and partially qualified domain names it is recommended to parse out 
+        the additional fields of top_level_domain, registered_domain, subdomain, and hostname
+        as they are available.
+      example: www.east.mydomain.co.uk 
+
+    # Domain Stuff
+    - name: hostname
+      level: core
+      type: keyword
+      short: Hostname of the system.
+      description: >
+        Hostname of the system.
+
+        It normally contains what the `hostname` command returns on the host machine, or the
+        host portion of a fully qualified domain name.
+        
+        For example, the hostname portion of "www.east.mydomain.co.uk " is "www".
+      example: www
```

Comment:
I'm not sure www is a good example of a hostname, especially outside of the url field set. See my comments on subdomain.

Reply:
Like you said, relates back to the subdomain, but I can go back and replace www with something less obviously a virtual name for the sake of the example, I think that makes sense.